src/modules/variables.js

```diff
@@ -0,0 +1,2 @@
+export const arrowBtn = document.getElementById('arrow');
+export const clearButton = document.getElementById('clear');
```

Comment:
- [ ] I think this file should contain methods related to status updates (completed: true / false). Please put the status update-related methods in this file instead of just variable declarations.